### ISSUE TITLE
Updating title for FixedPagingViewController

### ISSUE THREAD

**Issue by bizibizi** [rechsteiner/Parchment#200]
after updating title the selection disappears from menu items.

Here is the code: 

     let count = singerSongsSwitcherWireframe.presenter.dataSourceCount
        if count == 0 {
            singerSongsSwitcherUserInterface.title = "Songs".localized
        } else {
            singerSongsSwitcherUserInterface.title = "Songs".localized + " \(count)"
        }
        pagingViewController.reloadData()

//menu does not displayed that any item is selected

**Comment by bizibizi** [comment]
well, reloadData breaks everything :(

**Comment by rechsteiner** [comment]
Yeah, sorry about that. I just merged a fix into master #201. Could you target the master branch and see if it works now? 🙏

**Comment by bizibizi** [comment]
@rechsteiner so far everything works fine, thank you! 🙏

**Comment by eftalersoy** [comment]
I am getting same issue on latest version.
calling reload data on notificationcenter observer breaks evverything for FixedPagingViewController.

**Comment by rechsteiner** [comment]
@eftalersoy can you open a new issue with some more details? 🙂
